run-dask-process.sh

```diff
@@ -65,47 +65,104 @@ fi
 
 ########################################
 
-export DASK_UCX__CUDA_COPY=True
-export DASK_UCX__TCP=True
-export DASK_UCX__NVLINK=True
-export DASK_UCX__INFINIBAND=True  ###
-export DASK_UCX__RDMACM=True   ###
-export DASK_RMM__POOL_SIZE=0.5GB
-export DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT="100s"
-export DASK_DISTRIBUTED__COMM__TIMEOUTS__TCP="600s"
-export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MIN="1s"
-export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
-export DASK_DISTRIBUTED__WORKER__MEMORY__Terminate="False"
-
-#export DASK_UCX__REUSE_ENDPOINTS=False   ###
-export UCXPY_IFNAME="ib0"   ###
-#export UCX_NET_DEVICES=all   ###
-export UCX_MAX_RNDV_RAILS=1  # <-- must be set in the client env too!
-#export DASK_UCX_SOCKADDR_TLS_PRIORITY=sockcm   ###
-#export DASK_UCX_TLS=rc,sockcm,cuda_ipc,cuda_copy   ###
 export DASK_LOGGING__DISTRIBUTED="DEBUG"
 
 ulimit -n 100000
-export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7
 
-SCHEDULER_ARGS="--protocol=ucx
-                --port=8792
-                --interface=ib0
-                --scheduler-file $SCHEDULER_FILE
-               "
 
-WORKER_ARGS="--enable-tcp-over-ucx
-             --enable-nvlink 
-             --enable-infiniband
-             --enable-rdmacm
-             --rmm-pool-size=$WORKER_RMM_POOL_SIZE
+DASK_SCHEDULER_PORT=${DASK_SCHEDULER_PORT:-8792}
```

Comment:
Potentially move this to the `default-config.sh` script

Reply:
This is a good idea.